opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -231,6 +231,8 @@ impl trace::SpanExporter for Exporter {
             self.local_endpoint.clone(),
         ))
     }
+
+    fn set_resource(&mut self, _resource: &Resource) {}
```

Comment:
surprised that zipkin wasn't using Resources!

Reply:
Yes, it initializes the resource detectors during startup, and then it never changes. Will create a issue for this.

Reply:
Created an issue #1843